code/radar/radar.cpp

```diff
@@ -181,10 +181,10 @@ void HudGaugeRadarStd::drawContactCircle( int x, int y, int rad )
 			if (Missiontime & 8192)
 				return;
 		}
-		renderString( Large_blip_offset_x+x, Large_blip_offset_y+y, Large_blip_string );
+		renderCircle(x, y, 6);
 	} else {
 		// rad = RADAR_BLIP_RADIUS_NORMAL;
-		renderString( Small_blip_offset_x+x, Small_blip_offset_y+y, Small_blip_string );
+		renderCircle(x, y, 4);
```

Comment:
I made the choice to always use "real" circles since that it what's in the font files. These should look better than the font glyphs in all circumstances and I don't think there is a mod that uses special radar blip glyphs.

If there is a mod that uses special glyphs we can always change it back but this is, at least in my opinion, always an improvement over the current code so I chose to enable it always.